### PR TITLE
bump jruby 9.0.0.0-pre2

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: library/jruby [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+1.7-onbuild: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 1.7/onbuild
+1.7.19-onbuild: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 1.7/onbuild
 
-9.0.0.0: git://github.com/cpuguy83/docker-jruby@3a9e548c1f080e9116c13a8d62bf01ebc2e2c629 9000/jre
-9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@3a9e548c1f080e9116c13a8d62bf01ebc2e2c629 9000/jre
-9.0.0.0.pre1: git://github.com/cpuguy83/docker-jruby@3a9e548c1f080e9116c13a8d62bf01ebc2e2c629 9000/jre
-9.0.0.0.pre1-jre: git://github.com/cpuguy83/docker-jruby@3a9e548c1f080e9116c13a8d62bf01ebc2e2c629 9000/jre
+9.0.0.0: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
+9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
+9.0.0.0.pre2: git://github.com/cpuguy83/docker-jruby@540fa78ad4845773b0d9e361fa7f8000161825a1 9000/jre
```
Comment (tianon) [inline library/jruby]: Ah, looks like this one got missed somehow. :cry:

**Comment by cpuguy83** [comment]
Fixed.

**Comment by tianon** [comment]
LGTM (https://github.com/cpuguy83/docker-jruby/compare/3a9e548c1f080e9116c13a8d62bf01ebc2e2c629...0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b)

Build test of #702; bf12d7dcdc85aa34b1593787216c9c9f767ae019 (`jruby`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/bf12d7dcdc85aa34b1593787216c9c9f767ae019/library/jruby"
$ bashbrew build "$url"
Processing jruby:1.7 ...
Processing jruby:1.7.19 ...
Processing jruby:latest ...
Processing jruby:1.7-jre ...
Processing jruby:1.7.19-jre ...
Processing jruby:1.7-jdk ...
Processing jruby:1.7.19-jdk ...
Processing jruby:1.7-onbuild ...
Processing jruby:1.7.19-onbuild ...
Processing jruby:9.0.0.0 ...
Processing jruby:9.0.0.0-jre ...
Processing jruby:9.0.0.0.pre2 ...
Processing jruby:9.0.0.0.pre2-jre ...
Processing jruby:9.0.0.0-jdk ...
Processing jruby:9.0.0.0.pre2-jdk ...
Processing jruby:9.0.0.0-onbuild ...
Processing jruby:9.0.0.0.pre2-onbuild ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing jruby:1.7
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5]...passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
testing jruby:1.7-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5]...passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
testing jruby:1.7-onbuild
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5]...passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
testing jruby:9.0.0.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5].../opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCANNER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant PARSER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant Version
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCRIPT_LINES__
passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
testing jruby:9.0.0.0-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5].../opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCANNER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant PARSER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant Version
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCRIPT_LINES__
passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
testing jruby:9.0.0.0-onbuild
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'ruby-standard-libs' [3/5].../opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCANNER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant PARSER_EVENT_TABLE
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant Version
/opt/jruby/lib/ruby/stdlib/ripper/core.rb:13: warning: already initialized constant SCRIPT_LINES__
passed
	'ruby-gems' [4/5]...passed
	'ruby-bundler' [5/5]...passed
```

**Comment by tianon** [comment]
Those warnings in 9.0.0.0 are ugh, but meh; maybe jruby upstream bugs?

**Comment by cpuguy83** [comment]
Seems so.
Opened https://github.com/jruby/jruby/issues/2904 after speaking with jruby maintainers.

**Comment by yosifkit** [comment]
@cpuguy83 are we good to merge here?

**Comment by cpuguy83** [comment]
Yes, this is a pre-release build afterall :)

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**